src/base-components/Listing/BaseListing.js

```diff
@@ -268,16 +276,23 @@ export default {
             axios.get(this.url, options).then(response => this.populateCurrentStateAndData(response.data.data), error => {
                 // TODO handle error
             });
+
+            if(updateUrl) {
```

Comment:
keduy chces volat loadData s updateUrl=false?

Reply:
pouziva sa to teraz na event zmeny urlky cez back/forward button. Chces nacitat aktualne data, ale nechces menit url (lebo je uz zmenena)